examples/Makefile-da15000

```diff
@@ -0,0 +1,245 @@
+#
+#  Copyright (c) 2016, Dialog Semiconductor Ltd.
```

Comment:
Copyright must be "Copyright (c) 2016, The OpenThread Authors." for all files not in third_party.